script/generate-validation-schema.js

```diff
@@ -126,6 +128,11 @@ const onModelGetEnd = (model) => {
   })
 
   const namedEntities = objectsWithNameToNamedObjects(objects);
-
-  fs.writeFileSync('validationSchema.json', JSON.stringify(namedEntities));
+  onProcessed(namedEntities)
 }
+
+// run((schema) => {
+//   fs.writeFileSync("schema.json", JSON.stringify(schema));
+// })
```

Comment:
Czy ten kod zakomentowany jest potrzebny? Wybacz, że tak się czepiam tych komentarzy, ale zwykle to jest pozostałość po czymś.

Reply:
to co jest w folderze scripts i tak nie ląduje w node_modules jak się tego używa jako paczki, więc w sumie może zostać, jakby ktoś chciał odpalać ten skrypt z terminala z callbackiem